File: exercicios/implementacoes_csharp/ArvoreAvl/NoAvl.cs
namespace ArvoreAvl;

public class NoAvl : No 
{
    public int Fb {get; set;}

    public NoAvl(No? FilhoEsquerdo, No? FilhoDireito, No? Pai, int Elemento, int Fb) : base(FilhoEsquerdo, FilhoDireito, Pai, Elemento)
    {
        this.Fb = Fb;
    }
}

File: exercicios/implementacoes_java/trabalho_arvore/arvore_avl/NoAVL.java
package arvore_avl;
import arvore_binaria.NoArvore;

public class NoAVL extends NoArvore{
    private int fb;

    public NoAVL(NoAVL filhoEsquerdo, NoAVL filhoDireito, NoAVL pai, int o){
        super(filhoEsquerdo, filhoDireito, pai, o);
        this.fb = 0;
    }

    public int getFB(){
        return this.fb;
    }
}